pkg/envoy/server.go

```diff
@@ -199,7 +204,7 @@ func StartXDSServer(stateDir string) *XDSServer {
 										"prefix": {Kind: &structpb.Value_StringValue{StringValue: "/"}},
 									}}}},
 									"route": {Kind: &structpb.Value_StructValue{StructValue: &structpb.Struct{Fields: map[string]*structpb.Value{
-										"cluster":          {Kind: &structpb.Value_StringValue{StringValue: "cluster1"}},
+										// "cluster":          {Kind: &structpb.Value_StringValue{StringValue: "cluster1"}},
```

Comment:
Remove the commented line.

Reply:
I've left the comments here on purpose as placeholders for members that are added for a specific redirect. They help reminding that the lines below that actually set these members need to be adjusted if the filters are moved around etc.